### PR TITLE
Feature/plcn 404 create admin form

### PR DESCRIPTION
**Pull request by s-bogelund**
This includes both plcn 403 and 404.
An admin page has been implemented including routes and links in navbar.
The CreateUserForm uses the same logic as the backend to ensure password rules are being followed as well as ensuring the admin has typed the same password twice. If not, the user will get an appropriate error in an Alert.

### CODE REVIEW

File: src/components/admin/CreateUserForm.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+type CreateUserProps = {
+	onFormSubmit: (form: FormData) => void;
+	onPasswordMismatch: () => void;
+	onRuleError: () => void;
+} & FormGroupProps;
+
+const CreateUserForm: FC<CreateUserProps> = ({ onFormSubmit, onPasswordMismatch, onRuleError }) => {
+	const [password, setPassword] = useState('');
+	const [repeatedPassword, setRepeatedPassword] = useState('');
+	const [invalidPassword, setInvalidPassword] = useState(false);
```
Comment (itm-mne) [inline src/components/admin/CreateUserForm.tsx:25]: Could be prefixed with "is" to signify that it is a boolean and not the invalid password.

**Review by itm-mne — CHANGES_REQUESTED** [review]
Small rename - else looks fine

**Review by AntonEsbjergIT-Minds — APPROVED** [review]

**Review by itm-mne — APPROVED** [review]

File: src/components/login/LoginForm.tsx [hunk 2]
```diff
@@ -6,35 +6,38 @@ import {
...
-		// fire event to parent
+
+		onFormSubmit(data);
 	};
 
-	const isPasswordValid = useMemo(() => {
-		if (!failedLogin) return true;
-		// Password must be at least 8 characters long, contain at least one uppercase letter, one lowercase letter, and one number
-		const regex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$/;
+	//TODO: Move this logic to a create user page instead of login page
```
Comment (EsBjerrum) [inline src/components/login/LoginForm.tsx:33]: Det er vel flyttet, så skal det ikke bare slettes i stedet for udkommenteres?

File: src/pages/Login.tsx [hunk 3]
```diff
@@ -1,37 +1,47 @@
...
+					display: 'flex',
+					flexDirection: 'column',
+					alignItems: 'center',
+					width: '100%',
+				}}
+			>
+				<LoginForm onFormSubmit={form => handleLogin(form)} />
+			</Box>
+			<Snackbar
+				sx={{ width: '600px' }}
```
Comment (EsBjerrum) [inline src/pages/Login.tsx:35]: Måske lav en snackbarcomponent, der havde den rigtige styling og positionering, som kunne genbruges :)

**Review by EsBjerrum — APPROVED** [review]

**Pull request merged**